BackendNew/scripts/create-tables.sql

```diff
@@ -3,27 +3,26 @@
  Vacancy table, Student table, Student_Select_Company table, Monthly report table and Feedback table
  */
 CREATE TABLE Role (
-    RoleId int NOT NULL AUTO_INCREMENT,
     Name varchar(255) NOT NULL,
     Description varchar(255),
-    PRIMARY KEY (RoleId)
+    PRIMARY KEY (Name)
 );
 
 CREATE TABLE Admin (
     AdminId int NOT NULL AUTO_INCREMENT,
```

Comment:
Can we have auto incremented number as username?

Reply:
No I make it as a Name in the latest version